python/ray/experimental/data/tests/test_dataset.py

```diff
@@ -1089,32 +1089,6 @@ def test_csv_write(ray_start_regular_shared, tmp_path):
     shutil.rmtree(path)
 
 
-# TODO: this shouldn't be making network calls
-def test_uri_parser():
```

Comment:
Could we retain the coverage of this test?

Reply:
We no longer use `_uri_parser`, that function has been deleted. I've added a TODO to add a unit test to cover `_resolve_paths_and_filesystem`, but I don't think it's a high priority given that the CSV and JSON tests already cover far more cases than this unit test.